From 370654e004d780398ad0208ead92239ff9533287 Mon Sep 17 00:00:00 2001
From: yann_dm <yann.montmarin@gmail.com>
Date: Wed, 12 Oct 2022 14:19:22 +0200
Subject: [PATCH] tr1 not needed with c++>11

---
 src/graph/helper.cc | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git src/graph/helper.cc src/graph/helper.cc
index 11723f7..49d8646 100644
--- src/graph/helper.cc
+++ src/graph/helper.cc
@@ -41,8 +41,8 @@
 #include <hpp/util/debug.hh>
 #include <iterator>
 #include <pinocchio/multibody/model.hpp>
-#include <tr1/unordered_map>
-#include <tr1/unordered_set>
+#include <unordered_map>
+#include <unordered_set>
 
 #define CASE_TO_STRING(var, value) \
   ((var & value) ? std::string(#value) : std::string())
@@ -511,16 +511,16 @@ struct Result {
   ProblemSolverPtr_t ps;
   GraphPtr_t graph;
   typedef unsigned long stateid_type;
-  std::tr1::unordered_map<stateid_type, StateAndManifold_t> states;
+  std::unordered_map<stateid_type, StateAndManifold_t> states;
   typedef std::pair<stateid_type, stateid_type> edgeid_type;
   struct edgeid_hash {
-    std::tr1::hash<edgeid_type::first_type> first;
-    std::tr1::hash<edgeid_type::second_type> second;
+    std::hash<edgeid_type::first_type> first;
+    std::hash<edgeid_type::second_type> second;
     std::size_t operator()(const edgeid_type& eid) const {
       return first(eid.first) + second(eid.second);
     }
   };
-  std::tr1::unordered_set<edgeid_type, edgeid_hash> edges;
+  std::unordered_set<edgeid_type, edgeid_hash> edges;
   std::vector<std::array<ImplicitPtr_t, 3> > graspCs;
   index_t nG, nOH;
   GraspV_t dims;
-- 
2.25.1

